Quick Start Documentation/FPGA Addon Quick Start Guide.md

```diff
@@ -0,0 +1,282 @@
+
+<style>
+    h1{
+        color:#87dfd6;
+    }
+    h2{
+        color:#01a9b4;
+    }
+</style>
+
+
+
+
+# Overview
+
+This add-on runs a FPGA bitfile without needing to implement the normal VeriStand FPGA template. It allows the transfer of basic scalar data types inline with VeriStand's primary control loop (PCL). It also adds support for reading/writing waveforms with FPGA DMA channels.​
+
+<br>
+<br>
+
+# Repo location
+
+[Github](https://github.com/ni/niveristand-fpga-addon-custom-device)
+
+---
+<br>
+
+# Requirements
+
+These drivers must be installed on hte deployment computer/target to function:
+
+- NI VeriStand 2018 or later
+- NI RIO 18.0 or later
+---
+<br>
+
+# FPGA Specifics
+
+To use the addon, first create an FPGA program. Use the provided FPGA IP (discussed below) or custom logic. Put items of interest to front panel controls and indicators. Put waveforms of interest to DMA channels. __It is a requirement to the FPGA code to have an IRQ (address of 30) to control when the FPGA bitfile starts.__ This IRQ only needs to control when DMA read/write operations start. Any other code can start whenever it makes sense. Below is an example showing DMA, controls, and indicators.
+
+
+![Getting Started](./Images/image002.png)
+
+---
+
+
+# Supported Datatypes
+
+## Scalar Datatypes (Controls and indicators)
+
+- Bool
+- U8
+- U16
+- U32
+- U64
+- I8
+- I16
+- I32
+- I64
+- SGL
+- FXP +16.3
+- FXP +/-16.5
+- FXP +/-16.7
+- FXP +18.3
+- FXP +/-20.5
+- FXP +/-24.5
+- FXP +/-27.5
+- FXP +/-32.8
+- FXP +/-32.16
+- FXP +/-64.32
+
+**Note**: Enum​s are not supported. Conve​rt them to a U8, U16, or U32 control or indicator. **Clusters are not supported.** Even if the clusters are made up of supported types. Items can be grouped by naming them on the FPGA with a **_GroupName.SignalName_** format. Support for grouping can only be set on the initialization screen.
+
+---
+<br>
+
+## DMA Datatypes
+
+- U8
+- I8
+- U16
+- I16
+- U32
+- I32
+- U64
+- I64
+- SGL
+- FXP +/-32.8
+- FXP +/-32.16
+- FXP +/-64.32
+
+---
+**Note** The unsigned integer data types can be converted into a bitpacked Boolean array. This is discussed in detail below.
+
+<br>
+
+## Specialty IO
```

Comment:
typo, should be Speciality, that's how the submodule is named

Reply:
This is the name of the addon on Github: 

https://github.com/NIVeriStandAdd-Ons/FPGA-Addon-Speciality-IO/tree/master/Source - FPGA Addon Specialty IO

I am not sure that the name of the submodule folder in this case is relevant ...